llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -7915,22 +7915,27 @@ static void printCoreNoteLLVMStyle(const CoreNote &Note, ScopedPrinter &W) {
 template <class ELFT> void LLVMELFDumper<ELFT>::printNotes() {
   ListScope L(W, "Notes");
```

Comment:
Maybe this should be `NoteSections` or `NoteSectionList` to disambigutate? @jh7370's suggestion also seems fine, but this seems to be the convention we normally use: to name the list as the plural of the type being listed. I'm fine either way, as long as we're outputting something reasonable, though.

Reply:
I changed it to NodeSectionList. That seems like a pretty reasonable tag

Reply:
The existing `ListScope` variables use `*` instead of `*List`. I wonder whether `NoteSections` would be better.

Reply:
changed